packages/service-clients/azure-client/package.json

```diff
@@ -85,8 +85,9 @@
 		"start:tinylicious:test": "npx @fluidframework/azure-local-service > tinylicious.log 2>&1",
 		"test": "npm run test:realsvc",
 		"test:realsvc": "npm run test:realsvc:tinylicious",
-		"test:realsvc:local:run": "mocha --recursive \"lib/test/**/*.spec.*js\" --timeout 10000",
-		"test:realsvc:tinylicious": "start-server-and-test start:tinylicious:test 7070 test:realsvc:local:run",
+		"test:realsvc:tinylicious:report": "npm run test:realsvc:tinylicious",
+		"test:realsvc:tinylicious": "start-server-and-test start:tinylicious:test 7070 test:realsvc:tinylicious:run",
+		"test:realsvc:tinylicious:run": "mocha --recursive \"lib/test/**/*.spec.*js\" --exit",
```

Comment:
Without `--exit`, the tests hang at completion.  This implies something isn't getting cleaned up so it's not great, but this at least gets the tests running again.